src/background-view.cpp

```diff
@@ -45,7 +45,7 @@ extern "C"
 
 struct process
 {
-    nonstd::observer_ptr<wf::view_interface_t> view;
+    nonstd::observer_ptr<wf::view_interface_t> view = nullptr;
```

Comment:
Isn't this automatically null-initialized?

Reply:
Yes.